docs/user_guides/debugging/debugging_environment.rst

```diff
@@ -40,9 +39,10 @@ You need to set the following variable for debugging:
 
     export NUMBA_OPT=0
 
-You can use ``NUMBA_DPPY_DEBUGINFO`` instead of ``debug`` option.
-If set to non-zero, enable debug for the full application by setting the default value of the debug option in jit.
-Default value equals to the value of ``NUMBA_DEBUGINFO``.
+It is possible to enable debug for the full application by setting environment variable ``NUMBA_DPPY_DEBUGINFO=1``
```

Comment:
s/debug/debug mode/

Reply:
s/setting environment/setting the environment/

Reply:
Fixed